### PR TITLE
[ET-563] Add api for TMS estimation celery task status check.

### PR DESCRIPTION
**Pull request by ShanshanHe**

### CODE REVIEW

File: etabotsite/etabotapp/views.py [hunk 1]
```diff
@@ -369,12 +393,16 @@ def post(self, request, format=None):
...
         logging.debug('self.request.user: {}, self.username {}'.format(
-        self.request.user, self.request.user.username))
+            self.request.user, self.request.user.username))
+
+        post_data = {}
+        if request.body:
+            logging.debug('request.body: {}'.format(request.body))
+            post_data = json.loads(request.body)
 
-        post_data = json.loads(request.body.decode(encoding='utf-8'))
```
Comment (aradnaev) [inline etabotsite/etabotapp/views.py:377]: @ShanshanHe , can you help me understand why you removed ".decode(encoding='utf-8'))"? doesn't seem to work with our UI without decoding: 

"File "/Users/aradnaev/projects/etabot/pmp-shanshan/pmp/etabotsite/etabotapp/views.py", line 404, in post
    post_data = json.loads(request.body)
  File "/anaconda3/envs/etabot/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'"

**Comment by aradnaev** [comment]
@ShanshanHe , looks like sqlalchemy needs to be added to requirements.txt

File: etabotsite/etabotapp/views.py (hunk 1 above)
Comment (ShanshanHe) [reply etabotsite/etabotapp/views.py:377]: @aradnaev Please update your local python environment to python3.6, which is also the version we use to build our django server container as well. For python 3.6, json.loads() accepts bytes or bytearrays, so the decode is not needed: https://stackoverflow.com/questions/29780060/trying-to-parse-request-body-from-post-in-django

**Comment by ShanshanHe** [comment]
> @ShanshanHe , looks like sqlalchemy needs to be added to requirements.txt

@aradnaev  thank you for spotting this! Will add it!

**Review by aradnaev — APPROVED** [review]

**Pull request merged**